The following modules were not unloaded:
  (Use "module --force purge" to unload all):

  1) xalt/2.10.45

Inactive Modules:
  1) xalt


Currently Loaded Modules:
  1) gmp/6.2.1    4) zlib/1.2.11      7) openmpi/4.0.6
  2) mpfr/3.1.6   5) gcc/8.4.1        8) cuda/11.2.2
  3) mpc/1.1.0    6) numactl/2.0.14   9) modtree/gpu

Inactive Modules:
  1) xalt

 

/home/x-cli32/chaohan/anaconda3/envs/trl_env/lib/python3.9/site-packages/transformers/utils/hub.py:111: FutureWarning: Using `TRANSFORMERS_CACHE` is deprecated and will be removed in v5 of Transformers. Use `HF_HOME` instead.
  warnings.warn(
HTTP Error 429 thrown while requesting HEAD https://huggingface.co/datasets/trl-lib/tldr/resolve/main/README.md
Retrying in 1s [Retry 1/5].
HTTP Error 429 thrown while requesting HEAD https://huggingface.co/datasets/trl-lib/tldr/resolve/main/README.md
Retrying in 2s [Retry 2/5].
HTTP Error 429 thrown while requesting HEAD https://huggingface.co/datasets/trl-lib/tldr/resolve/main/README.md
Retrying in 4s [Retry 3/5].
HTTP Error 429 thrown while requesting HEAD https://huggingface.co/datasets/trl-lib/tldr/resolve/main/README.md
Retrying in 8s [Retry 4/5].
HTTP Error 429 thrown while requesting HEAD https://huggingface.co/datasets/trl-lib/tldr/resolve/main/README.md
Retrying in 8s [Retry 5/5].
HTTP Error 429 thrown while requesting HEAD https://huggingface.co/datasets/trl-lib/tldr/resolve/main/README.md
Traceback (most recent call last):
  File "/anvil/scratch/x-cli32/projects/trl/test_grpo.py", line 6, in <module>
    dataset = load_dataset("trl-lib/tldr", split="train")
  File "/home/x-cli32/chaohan/anaconda3/envs/trl_env/lib/python3.9/site-packages/datasets/load.py", line 1392, in load_dataset
    builder_instance = load_dataset_builder(
  File "/home/x-cli32/chaohan/anaconda3/envs/trl_env/lib/python3.9/site-packages/datasets/load.py", line 1132, in load_dataset_builder
    dataset_module = dataset_module_factory(
  File "/home/x-cli32/chaohan/anaconda3/envs/trl_env/lib/python3.9/site-packages/datasets/load.py", line 1027, in dataset_module_factory
    raise FileNotFoundError(
FileNotFoundError: Couldn't find any data file at /anvil/scratch/x-cli32/projects/trl/trl-lib/tldr. Couldn't find 'trl-lib/tldr' on the Hugging Face Hub either: LocalEntryNotFoundError: An error happened while trying to locate the file on the Hub and we cannot find the requested files in the local cache. Please check your connection and try again or make sure your Internet connection is on.
